tests/Data/ProgrammesDb/EntityRepository/CoreEntityRepository/FindProgrammesByAncestryAndTypesTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository::<public>
+ * @group repository
+ * @group programmesAggregation
+ */
+class FindProgrammesByAncestryAndTypesTest extends AbstractDatabaseTest
+{
+    public function testProgrammesChildrenGetCorrectClips()
+    {
+        $this->loadFixtures(['MongrelsFixture']);
+        $dbAncestryId = $this->getAncestryFromPersistentIdentifier('b010t19z', 'Brand');
+
+        $clipsUnderProgramme = $this->getEntityManager()
+            ->getRepository('ProgrammesPagesService:CoreEntity')
+            ->findProgrammesByAncestryAndType($dbAncestryId, 'Clip', 100, 0);
+
+        $this->assertCount(5, $clipsUnderProgramme);
```

Comment:
Check out using the `$this->assertEquals($expectedPids, array_column($entities, 'pid'));` style from the other tests, e.g. FindChildrenSeriesByParentTest.

That will save you testing each row of the results.